src/createBreakpoint.ts

```diff
@@ -1,7 +1,26 @@
 import { useEffect, useState, useMemo } from 'react';
 
+interface Screen {
+  laptopL: number;
+  laptop: number;
+  tablet: number;
+}
+
+enum ScreenSize {
+  LAPTOPL = 1440,
+  LAPTOP = 1024,
+  TABLET = 768,
+}
+
+const NAME = 0;
+const WIDTH = 1;
+
 const createBreakpoint = (
-  breakpoints: { [name: string]: number } = { laptopL: 1440, laptop: 1024, tablet: 768 }
```

Comment:
The idea of `{ [name: string]: number }` is that user can specify any number of breakpoints. I don't think we want to limit it to only `laptopL`, `laptop` and `tablet`.

Reply:
right! I didn't understand all your thoughts. 
that part reverted :)

Can you see pr?
I'll try to look into this repo more.